examples/filetimes.py

```diff
@@ -90,7 +90,7 @@ def benchmark(fn, args, filetype=None):
         if p.dftype == 'dask' and DD_FORCE_LOAD:
             if DEBUG:
                 print("DEBUG: Force-loading Dask dataframe", flush=True)
-            df = res.persist()
+            res = res.persist()
```

Comment:
So it does *not* use side effects?  I thought your comments above said it did.  Anyway, if it works as you say here, the code is fine.